### ISSUE TITLE
Unpublish package pycom.Pymakr

### ISSUE THREAD

**Issue by czadikem** [EclipseFdn/open-vsx.org#1584]
I did not sign the Publisher Agreement before I tried to publish this.  So the extension is inactive.

**Comment by czadikem** [comment]
I am also not the publisher of this software.

**Comment by kineticsquid** [comment]
Needs https://github.com/eclipse/openvsx/issues/664

**Comment by czadikem** [comment]
That does not allow me to remove this extension does it?

**Comment by kineticsquid** [comment]
@czadikem No, but I need it. See https://github.com/EclipseFdn/open-vsx.org/issues/1573#issuecomment-1412420641

**Comment by czadikem** [comment]
Oh ok thank you.👍

**Comment by czadikem** [comment]
@kineticsquid is this able to be unpublished yet?

**Comment by kineticsquid** [comment]
You should be good now.
